src/main/java/com/mixpanel/android/viewcrawler/EditProtocol.java

```diff
@@ -351,11 +358,11 @@ private Object convertArgument(Object jsonArgument, String type, List<String> as
         }
     }
 
-    private Drawable readDrawable(JSONObject description, List<String> assetsLoaded)
```

Comment:
I think choosing to interpret Drawable as a particular subclass was a mistake I made in the earlier protocol- the UI is smart enough to pick a concrete class (like BitmapDrawable or ColorDrawable) and it means that if we product NinepatchDrawables or ScaleDrawables or whatever in the future, it'll be a bit less confusing.